pkg/operator/vitesstopo/register.go

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
Let's call this file `register_cells.go` since it's a nice self-contained operation. Other operations added later should be their own file.

Reply:
Done https://github.com/planetscale/vitess-operator/pull/48/commits/02299a392aba6785fc0739748f5985d4cd3b1e92.